src/modules/enrollment/components/EnrollmentAssessmentActionButtons.tsx

```diff
@@ -55,35 +70,25 @@ const NewAssessmentMenu: React.FC<Props> = ({ enrollment }) => {
     const topItems: NavMenuItem[] = [];
     const bottomItems: NavMenuItem[] = [];
 
-    // Edge case: show "intake" item if the client is entered but does not have an intake
-    if (!enrollment.intakeAssessment) {
-      topItems.push({
-        key: 'intake',
-        to: getPath(AssessmentRole.Intake),
-        title: 'HUD Intake Assessment',
-      });
-    }
-
-    // Exit/Update/Annual can only be added to open enrollment
-    if (!enrollment.exitDate) {
-      topItems.push({
-        key: 'exit',
-        to: getPath(AssessmentRole.Exit),
-        title: 'HUD Exit Assessment',
+    if (!data?.client) return [];
+
+    const hudRoleTitles: Partial<Record<AssessmentRole, string>> = {
+      INTAKE: 'HUD Intake Assessment',
+      UPDATE: 'New HUD Update Assessment',
+      EXIT: 'New HUD Exit Assessment',
+      ANNUAL: 'New HUD Annual Assessment',
+      POST_EXIT: 'New HUD Post-Exit Assessment',
+    };
+
+    data.client.assessmentEligibilities.forEach(({ id, role, title }) => {
+      const items =
+        role === 'INTAKE' || role === 'EXIT' ? topItems : bottomItems;
```

Comment:
remove this, change them to be all in one section with no divider.


this was an attempt to separate out intake/exit as special types of assessments that can't have multiples. but it feels cluttered and unclear, so I think we should just simplify.